TaskManagement/registration/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from registration.views import Dashboard
+
+
+urlpatterns = [
+
+    path('', Dashboard.as_view(), name='dashboard'),
```

Comment:
it s better if you add path something like 'home/' instead of keeping null.

Reply:
check the latest commit